backstory/src/main/webapp/index.html

```diff
@@ -3,6 +3,15 @@
   <head>
     <meta charset="UTF-8" />
     <title>Home</title>
+
+    <!-- Bootsrap Stylesheet -->
```

Comment:
Nit: Might be good to including a link to the license in the comment. It's MIT licensed, which is fine, but it's hard to tell without inspecting the downloaded file right now:

https://github.com/twbs/bootstrap/blob/main/LICENSE

Reply:
Sounds good - updated!